pycalphad/tests/test_equilibrium.py

```diff
@@ -502,3 +543,22 @@ def test_eq_calculation_with_parameters():
                      {v.X('CU'): 0.0001052, v.P: 101325.0, v.T: 743.15, v.N: 1},
                      parameters=parameters, verbose=True)
     assert_allclose(eq.GM.values, -30374.196034, atol=0.1)
+
+
+@pytest.mark.solver
+def test_eq_alni_low_temp():
+    """
+    Low temperature Al-Ni keeps correct stable set at equilibrium.
+    """
+    dbf = NI_AL_DUPIN_2001_DBF
+    comps = ['AL', 'NI', 'VA']
+    phases = sorted(dbf.phases.keys())
+    eq = equilibrium(dbf, comps, phases, {v.P: 101325, v.T: 300, v.N: 1, v.X('AL'): 0.4})
+    # These values have NOT been verified in third-party software yet
```

Comment:
Checking the value should be something we can address. Let me know if you need me to check it.

Reply:
Can you check it?

Reply:
```
    assert_allclose(eq.GM.values, -63736.3048)
    assert_allclose(eq.MU.values.flatten(), [-116098.937755,  -28827.882809])
    assert set(np.squeeze(eq.Phase.values)) == {'BCC_B2', 'AL3NI5', ''}
    bcc_idx = np.nonzero(np.squeeze(eq.Phase.values) == 'BCC_B2')[0][0]
    al3ni5_idx = np.nonzero(np.squeeze(eq.Phase.values) == 'AL3NI5')[0][0]
    assert_allclose(np.squeeze(eq.X.sel(vertex=bcc_idx).values), [0.488104, 0.511896], atol=1e-6)
    assert_allclose(np.squeeze(eq.X.sel(vertex=al3ni5_idx).values), [0.375, 0.625], atol=1e-6)
```

Thermo-Calc values

```
 GM=-63736.305
MU(AL)=-116098.94, MU(NI)=-28827.882
NP(AL3NI5)=0.77896531 NP(BCC_B2)=0.22103469
X(BCC_B2,AL)=0.48810442 X(BCC_B2,NI)=0.51189558
X(AL3NI5,AL)=0.375 X(AL3NI5,NI)=0.625
```

LGTM